components/Team.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
```

Comment:
Discard changes to this file, already updated

Reply:
Fix that file